views/includes/documents/Terms-of-Service.md

```diff
@@ -38,15 +38,15 @@ Within moderation we understand the occasional necessity to conserve device stor
+* User Scripts have an implied license of [MIT License (Expat)](https://opensource.org/licenses/MIT) if the `@license` *(including the alternate spelling of `@licence`)* metadata block key is absent.
+* Library Scripts need to include one reference to their [acceptable license(s)](#acceptable-licensing) either via a License Header *([sample MPL header](https://www.mozilla.org/MPL/headers/) if that license is chosen)*, a cross-site compatible metadata block with at least one `@license` or `@licence` key **and** one `@exclude *`, or full License text *(full text is not recommended for storage constraints on portable devices)*. Libraries also have an implied license of [MIT License (Expat)](https://opensource.org/licenses/MIT) in the absence of any licensing *(however this should not happen)*. It may also be useful to indicate in the source that it is a library and not a userscript. If the type of script is unclear the library may be subject to removal.
 
 #### Acceptable Licensing
-* Licensing must be [Open Source Initiative *(OSI)* approved](http://opensource.org/licenses/category) and declared with the name of the License in the Source Code for all published works. e.g. `@license` or `@licence` keys, License Header or full License text *(full text is not recommended for storage constraints on portable devices)*
+* Licensing must be [Open Source Initiative *(OSI)* approved](https://opensource.org/licenses/category) and declared with the name of the License in the Source Code for all published works. e.g. `@license` or `@licence` keys, License Header or full License text *(full text is not recommended for storage constraints on portable devices)*
 * If a Content only license is included then the [Default Code Licensing](#default-code-licensing) will be applied with licensing arbitration, but may not always be visibly indicated on a script home page.
 * Any license that prohibits derivatives is eligible for removal.
 * Non-reusable licenses, of course, may not be used.
-* [Public domain type](http://opensource.org/faq#public-domain) licenses may mean different things in different jurisdictions and therefore are not allowed and eligible for removal.
+* [Public domain type](https://opensource.org/faq#public-domain) licenses may mean different things in different jurisdictions and therefore are not allowed and eligible for removal.
 * Any attempt to unlicense a published work will be eligible for removal.
 * In the event of any code licensing dispute resolution the [Default Code Licensing](#default-code-licensing) shall prevail if previously undeclared and/or undocumented.
 
```

Comment:
4 comments on commit df9d5b5

@Martii
Missing commit summary -1
@Martii
I won't bother doing the rest... but this shows you how we consider a PR.

@GreenLunar
No problems. Thank you very much.

@GreenLunar
To tell you the truth, I just wanted to raise this issue (if it is an issue at all), yet I did not want someone of the team members to wonder why they need to do despairing repetitive actions.

As far as I am concerned, I have done what I wanted to be done.